lib/okta.go

```diff
@@ -441,6 +452,12 @@ type OktaProvider struct {
 	ProfileARN      string
 	SessionDuration time.Duration
 	OktaAwsSAMLUrl  string
+	MFAConfig       MFAConfig
+}
+
+type MFAConfig struct {
```

Comment:
I know nothing else is really documented, but doc strings would be nice (examples!)